.github/workflows/cd-pypi.yml

```diff
@@ -10,6 +10,12 @@ on:
         type: boolean
         required: false
         default: false
+      work-directory:
+        description: Working directory to build Python package (for monorepos).
+          Defaults to root directory.
+        type: string
+        required: false
+        default: "/"
```

Comment:
Should this not be "./"? Otherwise we will 'cd' into the root directory of the file system if this is not supplied!

Reply:
Oh that's right!